src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -825,19 +837,18 @@ void SotaUptaneClient::runForever(const std::shared_ptr<command::Channel> &comma
         }
 
         sendImagesToEcus(updates);
-        *events_channel << std::make_shared<event::InstallComplete>();
 
         // FIXME how to deal with reboot if we have a pending secondary update?
-        boost::filesystem::path reboot_flag = "/tmp/aktualizr_reboot_flag";
-        if (boost::filesystem::exists(reboot_flag)) {
-          boost::filesystem::remove(reboot_flag);
-          if (getppid() == 1) {  // if parent process id is 1, aktualizr runs under systemd
-            exit(0);             // aktualizr service runs with 'Restart=always' systemd option.
-                                 // Systemd will start aktualizr automatically if it exit.
-          } else {               // Aktualizr runs from terminal
-            LOG_INFO << "Aktualizr has been updated and requires restarting to run the new version.";
-          }
-        }
+        // boost::filesystem::path reboot_flag = "/tmp/aktualizr_reboot_flag";
```

Comment:
Is it working without this hack now? If yes, we should delete this code.

Reply:
Reboot no. I think how to deal with it